go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
If you plan to maintain multiple versions of dependencies, consider using Go's `replace` directive to point to specific versions for testing or development purposes.